appnope/_nope.py

```diff
@@ -4,6 +4,10 @@
 #  Distributed under the terms of the 2-clause BSD License.
 #-----------------------------------------------------------------------------
 
+# Needed to init ObjC, otherwise C('NSString') returns None
+# TODO: Investigate why and include a better fix.
+from urllib.request import urlopen
```

Comment:
Instead of this line, we need to do `_ = ctypes.cdll.LoadLibrary(ctypes.util.find_library('Foundation'))` after the `objc` library load.  I think this should have been there all along (as the framework to interface with c code).